doc/conf.py

```diff
@@ -17,11 +17,11 @@
 # -- Project information -----------------------------------------------------
 
 project = 'DaCe'
-copyright = '2020, Scalable Parallel Computing Laboratory, ETH Zurich'
-author = 'Scalable Parallel Computing Laboratory, ETH Zurich'
+copyright = '2019-2020, Scalable Parallel Computing Laboratory, ETH Zurich'
+author = 'Scalable Parallel Computing Laboratory, ETH Zurich and the DaCe authors'
 
 # The full version, including alpha/beta/rc tags
-release = '0.9.5'
+release = '0.10.0a'
```

Comment:
Don't you want to do this separately?

Reply:
we could

Reply:
@definelicht but then I'd have to open a new PR for review :)